common/db_test.go

```diff
@@ -326,18 +326,73 @@ func TestSelectUpdateOrchs_AddingMultipleRows_NoError(t *testing.T) {
 	assert.Equal(orchsUpdated[1].ServiceURI, orchAdd.ServiceURI)
 }
 
+func TestOrchCount(t *testing.T) {
+	assert := assert.New(t)
+	var orchList []string
+	var nilDB *DB
+	zeroOrchs, nilErr := nilDB.OrchCount(&DBOrchFilter{})
+	assert.Zero(zeroOrchs)
+	assert.Nil(nilErr)
+
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	assert.Nil(err)
```

Comment:
Should we use `require` here since the rest of the tests would encounter problems if an error is encountered here?

Reply:
fixed in 2957c859